src/withResolution/index.js

```diff
@@ -11,7 +11,7 @@ import {
   hasTrueValues,
   type ShouldShow,
 } from './withResolution.logic'
-import errors from '../errors'
+import errors from /* preval */ '../errors'
```

Comment:
could we just be adding a `// @preval` comment to the top of errors.js?

Reply:
tests where choking on it, if the file itself gets preval'ed then it also doesn't count towards test coverage which it's not that big of a deal but it's unfair because it is tested

Reply:
hm maybe it's something webpack should do, alias `../errors` to something like `../errors/preval` (which imports with preval and exports) and then tests would remain unaffected

Reply:
Eh that seems more confusing and magical. Would we be able to disable preval for tests?

Reply:
yeah it's a balancing act, with the current approach if you forget to import `./errors` without preval you'll import it twice (and increase bundle size), if we do it the other way there's some webpack magic.

I'm fine either way, I don't like either approach tbh

We can disable preval for tests _if_ nothing else is using preval

Reply:
I'm fine keeping it this way. Since bundlesize should catch mistakes like that.